src/debug-macros.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-env node */
+'use strict';
+const requireEsm = require('esm')(module);
+const semver = require('semver');
+
+function gte(availableVersion, compatVersion) {
+  return semver.gte(semver.minVersion(availableVersion), semver.minVersion(compatVersion));
+}
+
+function getPackage(dep, packageName, seen) {
```

Comment:
I didn't see tests for the functions in this module, can you point me to them? If they don't exist, they should be implemented.

Reply:
Added tests and made them run in travis